src/index.html

```diff
@@ -1,12 +1,396 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page_body">
+
+    <header class="header">
```

Comment:
```suggestion
  <body class="page_body">
    <header class="header">
```
Don't leave empty lines between parent and child elements